<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFloatMarkerWidget</class>
 <widget class="QWidget" name="EditFloatMarkerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="intervalButton">
     <property name="text">
      <string>Interval:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="intMinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="maximum">
        <double>400000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="intMaxBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="maximum">
        <double>400000000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="lessButton">
     <property name="text">
      <string>Less than:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="lessBox">
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="maximum">
      <double>400000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>66</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="greaterButton">
     <property name="text">
      <string>Greater than:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="greaterBox">
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="maximum">
      <double>400000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>66</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>intervalButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>intMinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>intMaxBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lessButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>lessBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greaterButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>greaterBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
